src/tools/wasm-emscripten-finalize.cpp

```diff
@@ -249,14 +249,18 @@ int main(int argc, const char* argv[]) {
     BYN_TRACE("finalizing as side module\n");
     PassRunner passRunner(&wasm);
     passRunner.add("replace-stack-pointer");
+    passRunner.add("emscripten-pic-abi");
     passRunner.run();
     generator.generatePostInstantiateFunction();
   } else {
     BYN_TRACE("finalizing as regular module\n");
+    PassRunner passRunner(&wasm);
+    passRunner.add("emscripten-pic-abi-main-module");
+    passRunner.run();
```

Comment:
separate from this PR, if we added a return from `add()` then these one-pass runnings could be
```cpp
PassRunner(&wasm).add("foo").run();
```
WDYT?

Reply:
Ive got a refactor coming up as followup the unifies all of this into on pass runner.  So it will not longer be needed here.